File: Examples/VisualBasic/AsposePdfGenerator/Working-Document/SetAppearance.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports AP = Aspose.Pdf
Namespace AsposePdfGenerator.WorkingDocuments
    Public Class SetAppearance
        Public Shared Sub Run()
            ' ExStart:SetAppearance
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdfGenerator_WorkingDocuments()

            ' Instantiate Pdf instance
            Dim pdf1 As New AP.Generator.Pdf()

            ' Set OpenType property of Pdf instance to any pre-defined value
            pdf1.OpenType = AP.Generator.OpenType.Thumbnails

            dataDir = dataDir & Convert.ToString("SetAppearance_out_.pdf")
            pdf1.Save(dataDir)
            ' ExEnd:SetAppearance
            Console.WriteLine(Convert.ToString(vbLf & "Appearance setup successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePdfGenerator/Working-Document/SetAppearance.cs
using System;
using System.IO;
using Aspose.Pdf;

namespace Aspose.Pdf.Examples.CSharp.AsposePdfGenerator.WorkingDocuments
{
    public class SetAppearance
    {
        public static void Run()
        {
            // ExStart:SetAppearance
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdfGenerator_WorkingDocuments();

            // Instantiate Pdf instance
            Aspose.Pdf.Generator.Pdf pdf1 = new Aspose.Pdf.Generator.Pdf();

            // Set OpenType property of Pdf instance to any pre-defined value
            pdf1.OpenType = Aspose.Pdf.Generator.OpenType.Thumbnails;

            dataDir = dataDir + "SetAppearance_out.pdf";
            pdf1.Save(dataDir);
            // ExEnd:SetAppearance
            Console.WriteLine("\nAppearance setup successfully.\nFile saved at " + dataDir);
        }
    }
}